changelog.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Move ELK parameters from ElkLayout to DefautConfiguration.
+- Add `repositionComponent` method in `DefaultDrawer`, `DefaultPlugin` and `DefaultLayout`.
```

Comment:
I see it now, but place in wrong section, move to `### Added`

Reply:
Oops, there was a conflict while rebasing and I messed this.
Corrected.